servicebus/data-plane/azure-servicebus/pom.xml

```diff
@@ -0,0 +1,99 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Should the file above be deleted? azure-servicebus.pom?
The spacing in this is off.

Reply:
I think azure-servicebus.pom can't be deleted but should renamed to pom.xml. Because it is the module pom of service bus parent pom. However, I am not 100% sure since there is only one child module pom file. @JonathanGiles

Reply:
There are two files in the same directory, one called azure-servicebus.pom and another pom.xml. It threw me off.

servicebus/data-plane/azure-servicebus/azure-servicebus.pom
servicebus/data-plane/azure-servicebus/pom.xml

Reply:
Sorry that I don't catch it at the first place. Yes. It is a duplicate pom file and deleted